contrib/helm/harbor/README.md

```diff
@@ -189,4 +189,23 @@ The following tables lists the configurable parameters of the Harbor chart and t
 | `notary.affinity` | Node/Pod affinities | `{}` |
 
 ## Persistence
-TBD
+
+You need to create `StorageClass` before able to persist data in persistent volume.
```

Comment:
You need to create `StorageClass` before you can persist data in persistent volume.